server/clip_server/executors/clip_torch.py

```diff
@@ -52,6 +52,7 @@ def __init__(
 
         self._pool = ThreadPool(processes=num_worker_preprocess)
 
+    @monitor('preproc_images_seconds','Time preprocessing images')
```

Comment:
Does the monitor decorator work with arbitrary function? or it can only work with the Executor function (i.e., member function of the executor) whose input and outputs are `DocumentArray`?

Reply:
It works with any method of the Executor, it does not require any particular signature. It still need to be a method and not a arbitrary function though, as it needs to access the metrics registry which is store inside the Executor

Reply:
I see.